app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/AllSessionsFragment.kt

```diff
@@ -113,6 +118,28 @@ class AllSessionsFragment :
         binding.sessionsRecycler.smoothScrollToPosition(0)
     }
 
+    override fun saveCurrentSession() {
+        val layoutManager = binding.sessionsRecycler.layoutManager as LinearLayoutManager
+        val scrollState = layoutManager.getScrollState()
+        Prefs.previousSessionScrollPosition = scrollState.anchorPosition
```

Comment:
Can you create a property in PreviousSessionPrefs?

```
val scrollState: ScrollState
  get() = ScrollState(anchorPosition, anchorOffset)
  set() = anchorPosition = state.anchorPosition...
```

Reply:
Okay, I see. 
I try to create.